kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationJobAPIImpl.java

```diff
@@ -266,7 +294,13 @@ private void dropCheckJobs(final String jobId) {
         log.info("dropCheckJobs start...");
         long startTimeMillis = System.currentTimeMillis();
         for (String each : checkJobIds) {
-            dropJob(each);
+            try {
+                dropJob(each);
+                // CHECKSTYLE:OFF
+            } catch (final RuntimeException ex) {
+                // CHECKSTYLE:ON
+                log.info("drop check job failed, check job id: {}, error: {}", each, ex.getMessage());
```

Comment:
why we use info level to print out the catched exception ?

Reply:
Since it could be ignored, just record it here without stack trace

Reply:
> Since it could be ignored, just record it here without stack trace

I see. then that exception instance ex should be called `ignored`.

> [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) `Exceptions when catch are named with ex; Exceptions when catch but do nothing are named with ignored`

Reply:
But it's used in logging, if `ignored` is used, it shows warning in IDEA:
<img width="896" alt="图片" src="https://user-images.githubusercontent.com/42492540/195232159-80c929c8-d089-4726-bb12-3637c61ecfad.png">
The ignored case:
<img width="413" alt="图片" src="https://user-images.githubusercontent.com/42492540/195232358-bf9f038e-d676-4d25-8a61-9f5f9e34f95e.png">